be/src/storage/sstable/merger.cpp

```diff
@@ -175,8 +175,6 @@ Iterator* NewMergingIterator(const Comparator* comparator, Iterator** children,
     assert(n >= 0);
     if (n == 0) {
         return NewEmptyIterator();
-    } else if (n == 1) {
```

Comment:
Why do u remove this line?

Reply:
We need to wrap iterator, so they can be released.